target/scripts/build/packages.sh

```diff
@@ -0,0 +1,118 @@
+#! /bin/bash
+
+# -eE :: exit on error (do this in functions as well)
+# -u  :: show (and exit) when using unset variables
+# -o pipefail :: exit on error in pipes
+set -eE -u -o pipefail
+
+# shellcheck source=../helpers/log.sh
+source /usr/local/bin/helpers/log.sh
+
+QUIET='-qq' ; [[ ${LOG_LEVEL} =~ ^trace$ ]] && QUIET='-y'
```

Comment:
Yes, this will just increase log verbosity. We need the `-y` flag I think because the whole process is non-interactive (not sure whether `DEBIAN_FRONTEND=noninteractive` is enough). I will test and adjust.